partcad-cli/src/partcad_cli/click/commands/system/set/telemetry/env.py

```
@@ -0,0 +1,48 @@
+            yaml.preserve_quotes = True
+            if not os.path.exists(config_path):
+                Path(config_path).touch()
+                config = {}
+            else:
+                with open(config_path) as fp:
+                    config = yaml.load(fp)
+                    fp.close()
+
+            if not config:
+                config = {}
+            if not "telemetry" in config:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)